app/controllers/taxonomy_signups_controller.rb

```diff
@@ -0,0 +1,43 @@
+class TaxonomySignupsController < ApplicationController
+  def new
+    redirect_to '/' and return unless params[:paths].present?
+
+    taxon_path = params[:paths].first
+    load_taxon(taxon_path)
```

Comment:
Breaks noisily if it can't find a taxon with the given base path, which is reasonable i think.